firmware/controllers/sensors/sensor_type.h

```diff
@@ -113,6 +113,13 @@ enum class SensorType : unsigned char {
 
 	DetectedGear,
 
+	RangeInput1,
```

Comment:
does it make sense to put the un-decoded range inputs in the sensor model, or to put only the "decoded" PRNDL21whatever?

Reply:
It looked to me like the way to read an analog pin was to create a Sensor for it. Is there a better way?

Reply:
Also I found ResistanceFunc and wanted to use it.

Reply:
Andrey's suggestion was to reuse the Lua aux analog input Sensors, but that seemed like a bad idea to me.

Reply:
> Andrey's suggestion was to reuse the Lua aux analog input Sensors, but that seemed like a bad idea to me.

that was suggested only as a dev time shortcut